src/main/java/br/com/fiap/tech_challenge/adapters/driver/api/controller/OrderController.java

```diff
@@ -0,0 +1,32 @@
+package br.com.fiap.tech_challenge.adapters.driver.api.controller;
+
+import br.com.fiap.tech_challenge.adapters.driver.api.dto.CreateOrderRequestDTO;
+import br.com.fiap.tech_challenge.adapters.driver.api.dto.CreateOrderResponseDTO;
+import br.com.fiap.tech_challenge.adapters.driver.api.mapper.OrderMapper;
+import br.com.fiap.tech_challenge.core.domain.usecases.order.CreateOrderUseCase;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/orders")
```

Comment:
@DiguyaDeveloper Aqui daria para remover o /api ja que esta na raiz da aplicação

Reply:
Ela não está na raiz da aplicação ainda, ainda não foi feito esse ajuste no projeto

Acredito que assim que isso for feito, já retirmos de todas as controllers o basePath

Reply:
@JessicaRodriguesdc @DiguyaDeveloper Isso ta no PR do @8bitbeard quando mergearmos o dele, a gente ajusta em todos os controllers